share/shwap/p2p/bitswap/bitswap.go

```diff
@@ -132,6 +132,12 @@ func NewClient(
 		// These two options have mixed up named. One should be another and vice versa.
 		client.ProviderSearchDelay(broadcastDelay),
 		client.RebroadcastDelay(delay.Fixed(provSearchDelay)),
+		client.BroadcastControlEnable(true),
```

Comment:
They added some broadcast optimizations that I enabled
Initally it failed all the relevant units, but tunong configuration a bit made it work fine